app/scripts/views/mixins/inter-tab-signin-mixin.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'lib/constants',
+  'lib/promise',
+  'views/mixins/inter-tab-channel-mixin'
+],
+function (Cocktail, Constants, p, InterTabChannelMixin) {
+  'use strict';
+
+  var EVENTS = Constants.EVENTS;
+
+  return Cocktail.mixin({
+    afterVisible: function () {
+      this.navigateToSignedInView = navigateToSignedInView.bind(this);
+      this.interTabOn(EVENTS.SIGNIN_SUCCESS, this.navigateToSignedInView);
+    },
+
+    onSignInSuccess: function (account) {
+      this.interTabOff(EVENTS.SIGNIN_SUCCESS, this.navigateToSignedInView);
```

Comment:
If it does send the message back to the sender, should we update `lib/channels/inter-tab.js` to suppress that behavior? 

We locally host crosstab because of other changes we have made, maybe we comment out [this line in crosstab](https://github.com/mozilla/fxa-content-server/blob/92425c5e0bf72779b9d51d447fcf47ddc2dad2df/app/scripts/vendor/crosstab.js#L410)